src/components/SidebarItem/__test__/sidebarItem.spec.js

```diff
@@ -36,6 +36,22 @@ describe('<SidebarItem when href is passed />', () => {
         sidebarLink.simulate('click');
         expect(handleOnClickMockFn).toHaveBeenCalledTimes(1);
     });
+    it('should render the right icon when the item is selected', () => {
+        const component = mount(
+            <Sidebar selectedItem="sidebaritem-test-4">
+                <SidebarItem
+                    href="/#/Components"
+                    icon={<FontAwesomeIcon icon={faUser} />}
+                    selectedIcon={<FontAwesomeIcon icon={faClock} />}
+                    name="sidebaritem-test-4"
+                    label="sidebaritem-test-4"
+                />
+            </Sidebar>,
+        );
+        const sidebarLink = component.find('a');
+        sidebarLink.simulate('click');
+        expect(component.find({ 'data-icon': 'clock' }).exists()).toBe(true);
```

Comment:
This is something internal from FontAwesome? If true then we need to change this assertion since we can't make introspection inside what we could not control, font awesome can change this behaviour and then if we update then our test take the risk to fail, instead, we should find the FontAwesomeIcon component in the tree of react components and check that the icon prop passed is faClock